plugins/tekton/README.md

```diff
@@ -1,13 +1,12 @@
 # Tekton plugin for Backstage
 
-This plugin will help with visualizing the workloads like Deployment, Replicaset and Pods powering any service on the Kubernetes cluster.
+The Tekton plugin helps with visualizing the workloads such as Deployment, Replicaset, and Pods powering any service on the Kubernetes cluster.
```

Comment:
```suggestion
The Tekton plugin helps with visualizing the `PipelineRun` resources available on the Kubernetes cluster.
```

Reply:
What about TaskRuns should those be mentioned too?

Reply:
As of today, we are visualizing only the pipelineRun, but we fetch the taskRuns of the associated pipeline to show the task status. Maybe frame it in a way so that users do not get the impression that they can visualize the task runs like how they do with Pipeline Runs.

Attaching the screenshot of the PipelineRun list page for ref:

<img width="1547" alt="PR" src="https://user-images.githubusercontent.com/22490998/233917164-6ba363fc-e97c-4c71-a5f7-ea53cca0759d.png">

Reply:
Yeah needed task status as well to show the column task status bar viz

we do get taksStatus in the status of pipeline run status as well but we needed to fetch it separately as last I heard from tekton team is that task status in status of pipeline runs is getting depreciated so good to not rely on it.

Reply:
Done